### ISSUE TITLE
T-Rex pose not scientifically accurate

### ISSUE THREAD

**Issue by kevinranks** [emojione/emojione-assets#29]
from @salty-horse @ issue [#536](https://github.com/emojione/emojione/issues/536) in emojione repo.

> The EmojiOne image for U+1F996 T-REX shows it nearly upright. While this is a popular look for the dinosaur, it does not reflect current scientific understanding. "Canonizing" it in an emoji will help perpetuate the myth.
> 
> For more info:
> https://en.wikipedia.org/wiki/Tyrannosaurus#Posture
> http://news.cornell.edu/stories/2013/02/blame-barney-students-perception-t-rex-outdated

**Comment by thinkrick** [comment]
I completely agree with this.  We'll work on a major upgrade to our unscientific T-Rex for the next design update.  Thx for the feedback!  👍🏼

**Comment by neekless** [comment]
Seems like you forgot about this in the 4.0 [update](https://drive.google.com/open?id=1L2jwzGcIwaIZ5LU1jncGkjpvbI5kbZGH. The t-rex is still almost upright, when the correct posture should be the body almost parallel to the ground, like [this](https://drive.google.com/open?id=1wh0DOkQTGPTPYx5LbwxIOQh0Pn10yRAw).
